src/common/helpers.ts

```diff
@@ -66,3 +66,18 @@ export function hasAdditionalProperties(obj: ObjectSchema): DictionarySchema | u
   }
   return undefined;
 }
+
+export function isScalar(schema: Schema): boolean {
```

Comment:
Will this include enums and arrays of enums?

Reply:
Added Choice and SealedChoice schema type to the switch. I was also wondering about time types and time arrays....should those also have tags?

Reply:
I also saw Odata types...what does that mean? And should they be added?

Reply:
For track 1 it was any  primary type.  The track 1 modeler defines primary types [here](https://github.com/Azure/autorest.common/blob/04b5593e34604e993bdda8467def411849f5b90b/src/Model/KnownPrimaryType.cs).  So yeah the time types will need to be added too (possibly others, I didn't look closely at both lists).

Reply:
Ok I added all the schema types I think are applicable to us in isScalar.